developer-workflow/psrt.rst

```diff
@@ -68,24 +68,24 @@ severity, advisory text, and fixes.
   * Affected components and APIs. The module, function, class, or method must be specified so users can
     search their codebase for usage. For issues affecting the entire project, this can be omitted.
 
-  * Mitigations for the vulnerability beyond upgrading to a patched version, if applicable.
+  * Mitigations for the vulnerability beyond upgrading to a fixed version, if applicable.
```

Comment:
Just something to keep in mind: "fix" implies an initial "brokenness", which might work in this case, but doesn't necessarily work for e.g. doc changes.  Here "patched" works equally well IMHO, since it's more generic.

Reply:
To my ears, "a patched version" sounds like it might be locally or temporarily patched.  I think "fixed" works well here where are are talking about security vulnerabilities.